subgraphs/_reference_/src/sdk/protocols/generic/protocol.ts

```diff
@@ -0,0 +1,250 @@
+import { Versions } from "../../../../../../deployment/context/interface";
+
+import { Protocol as ProtocolSchema } from "../../../../generated/schema";
+import { TokenPricer } from "../config";
+import { Config } from "./config";
+import { ProtocolSnapshot } from "./protocolSnapshot";
+import { AccountWasActive } from "./account";
+import * as constants from "../../util/constants";
+import {
+  dataSource,
+  Address,
+  Bytes,
+  BigDecimal,
+} from "@graphprotocol/graph-ts";
+import { SDK } from ".";
+import { BIGINT_ZERO } from "../lending/constants";
+import { CustomEventType } from "../../util/events";
+
+/**
+ * Bridge is a wrapper around the BridgeProtocolSchema entity that takes care of
```

Comment:
“Bridge” :D 

You have a few references to Bridge stuff around

Reply:
Fixed!